hw5/task2/library.js

```diff
@@ -0,0 +1,184 @@
+const { setImmediate, setInterval } = require('timers');
+
+class Library{    
+    Track = [];
+    Album = [];
+    Artist = [];
+ // ADD METHOD WITH NESTINGS   
+    add(){ 
+    //    value.constuctor.name; - return Class name "Track,Artist,Album";
+    //    value.name - return class.name "Linkin park exemple";
+    // IF :for classes! if one value - this CLASS. if more value: first - Class, nexts - parameters this Class;
+    //ELSE: For class parameters. If we have more then one parameters = > [new Class,...parametrs for this class ];  
+                            // TRACK
+            if(arguments[0].constructor.name === "Track"){
```

Comment:
`arguments[0] instanceOf Track` is better to undestanding

Reply:
`arguments` are unusable since it is hard to understand what happens in this method without reading the implementation. Better define the method parameter and check this out.